Annulled configuration tick rate dependency

lamarrr opened the pull request:
Noticed this in the Google I/O 2019 talk's code snippet.
vTaskDelay actually accepts the number of ticks to delay for and not time in ms.
The time is dependent on the tick rate.

lamarrr commented:
I signed it!

lamarrr commented:
Message

bukepo commented:
@lamarrr I think there are some issues about the author of some commits of this PR. @gjc13 could you please share more help?

lamarrr commented:
Changed my github email address. And the email registered on my git config differs from my github's.
Sorry for the confusion. 
Moved here:  https://github.com/openthread/ot-rtos/pull/19
